ember_debug/vendor/loader.js

```diff
@@ -1,5 +1,10 @@
-let define = window.define,
+/* eslint-disable */
+let define = window.define && ((...args) => window.define(...args)),
+  exports = undefined,
+  module = undefined,
```

Comment:
Why set these to undefined?

Reply:
so that global exports, module etc will not make it into ember_debug